CHANGELOG.md

```diff
@@ -1,10 +1,40 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-10-15
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
```

Comment:
[CVE-2023-29332](https://github.com/Azure/AKS/issues/3904) Please update your AKS VHD to at least VHD version 230801 as mentioned in the issue.